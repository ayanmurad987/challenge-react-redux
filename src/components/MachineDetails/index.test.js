import React from 'react'
import { shallow, mount } from 'enzyme'
import MachineDetails from './index'
import store from '../../store'
import { Provider } from 'react-redux'

it('renders MachineDetails component without crashing.', () => {
  const wrapper = shallow(
    <Provider store={store}>
      <MachineDetails />
    </Provider>)
  expect(wrapper).toMatchSnapshot()
})
