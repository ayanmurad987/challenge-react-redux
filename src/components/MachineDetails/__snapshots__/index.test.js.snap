// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders MachineDetails component without crashing. 1`] = `ShallowWrapper {}`;
